packages/regl-worldview/src/commands/GLText.js

```diff
@@ -228,12 +258,12 @@ const frag = `
     // depth test don't work together nicely for partially-transparent pixels.
     float edgeStep = smoothstep(1.0 - cutoff - fwidth(dist), 1.0 - cutoff, dist);
 
-    if (scaleInvariant && vBillboard == 1.0 && scaleInvariantSize < 0.03) {
-      // If scale invariant is enabled and scaleInvariantSize is "too small", do not interpolate 
-      // the raw distance value since at such small scale, the SDF approach causes some 
+    if (scaleInvariant && vBillboard == 1.0 && scaleInvariantSize <= 20.0) {
```

Comment:
this is a pretty big bump from 0.03-- why the change?

Reply:
It's higher because this new approach does not normalize `scaleInvariantSize` by dividing it by the screen height before sending it to the shaders. The end end result is the same as before, to avoid using SDF if the character size is too small (less than 20 pixels).